randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "Gravity Suit.\nA harder method uses &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; to scan-dash"
+            " off of pickups left by the crates.\n\nFinally, it is possible to get across with "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; by L-jumping between the lips of the "
+            "leftmost pillars.\nThis is more difficult than it appears, as the closest pillar is not standable."
+        ),
+    ],
+    "455264308": [
+        # No Grappling/Spidering Thru Damage Tip
+        # Replaces Meditation Fountain panel scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Taking damage will interrupt use of the Spider Ball or Grapple Beam. "
+            "This includes damage from superheated rooms."
+        ),
+    ],
+    "2410883438": [
+        # Just Hold L Tip
+        # Replaces Main Plaza bird image scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; grants greater "
+            "control over aerial movement, making precise jumps easier."
+        ),
+    ],
+    "389886815": [
+        # Just Tap B Tip
+        # Replaces Observatory left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When performing a scan dash or lock-on dash, jump by tapping "
+            "&image=SI,0.70,0.68,833BEE04; for as little time as possible to gain extra height."
+        ),
+    ],
+    "3668253850": [
+        # Item Drop Tip
+        # Replaces Waterfall Cavern hole scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Each room has different pickups that drop from crates and defeated enemies.\n\n"
+            "If a specific pickup does not seem to be appearing (e.g. Power Bomb ammo), try a different room."
+        ),
+    ],
+    "3673769160": [
+        # L-Jump Frames Tip
+        # Replaces Elite Research top floor computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; for the first 13 frames of a jump "
+            "and then letting go\nslightly extends jumps. This is called an L-Jump. When using the &main-color=#FF3333;"
+            "Space Jump Boots&main-color=#89D6FF;, holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; "
+            "for 7 frames after the second jump grants an additional, smaller extension."
+        ),
+    ],
+    "1807281950": [
+        # RDV FAQ Tip
+        # Replaces North Atrium right sculpture scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;Each game in Randovania has a Frequently Asked Questions section.",
```

Comment:
This scan gives information outside the scope of the game. The FAQ is already visible and available on the Randovania application, so I do not see the point of it.

Reply:
Goal 2, and for this specifically, I've seen too many people obviously not know the FAQ exist to assume people notice it frequently. It's not like there's something else competing for this spot, so I think a reminder like this is useful to cut down on the number of times people ask something in #prime-help and get answered by the FAQ bot.